src/board/audio.ts

```diff
@@ -76,7 +114,8 @@ class BufferedAudio {
       buffer,
     });
     source.onended = this.callback;
-    source.connect(this.context.destination);
+    this.gainNode.connect(this.context.destination);
```

Comment:
Do we do this twice? Would it make more sense to pass in the pre-connected gain node as the logical destination for the buffered audio?